src/it/undefined-java-level/postbuild.groovy

```diff
@@ -0,0 +1,3 @@
+assert new File(basedir, 'build.log').text.contains('java.lang.IllegalArgumentException: Unknown JDK version given. Should be something like "1.7"')
```

Comment:
@batmat’s suggestion of [using `requirePropertyDiverges`](http://www.mojohaus.org/extra-enforcer-rules/requirePropertyDiverges.html) does not actually work—another rule bombs out first.

Reply:
A priori, `requierePropertyDiverges` sounds like the right direction.